cpp/unreal_plugins/SimulationController/Source/SimulationController/NavMesh.cpp

```diff
@@ -83,3 +90,72 @@ std::vector<uint8_t> NavMesh::getRandomPoints(int num_points)
     }
     return Std::reinterpretAs<uint8_t>(points);
 }
+
+std::vector<uint8_t> NavMesh::getReachablePoints(const std::vector<std::vector<float>>& start_points)
+{
+    std::vector<double> end_points;
+    for (auto& start_point : start_points) {
+
+        SP_ASSERT(start_point.size()==3);
+
+        FVector start_point_fvector{start_point.at(0), start_point.at(1), start_point.at(2)};
+        FNavLocation end_location;
+
+        bool found = recast_nav_mesh_->GetRandomReachablePointInRadius(
```

Comment:
The radius should be passed as input to this function, remove the config parameter from `default_config`.

Reply:
Updated this.